CHANGELOG.md

```diff
@@ -2115,8 +2115,8 @@ The following sections list the changes for 1.0.0.
 * Enhancement - Update konnectd to v0.33.8: [#744](https://github.com/owncloud/ocis/pull/744)
 
    This update adds options which allow the configuration of oidc-token expiration parameters:
-   KONNECTD_ACCESS_TOKEN_EXPIRATION, KONNECTD_ID_TOKEN_EXPIRATION and
-   KONNECTD_REFRESH_TOKEN_EXPIRATION.
+   IDP_ACCESS_TOKEN_EXPIRATION, IDP_ID_TOKEN_EXPIRATION and
```

Comment:
Please do not change the changelog. Should also apply to unreleased section of the changelog?

Reply:
Sorry those slipped trough :)